sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -247,7 +261,12 @@ def get_account_properties(self, **kwargs):
                 :caption: Get properties for the form recognizer account.
         """
         response = self._client.get_custom_models(error_map=error_map, **kwargs)
-        return AccountProperties._from_generated(response.summary)
+        try:
```

Comment:
try/catch should surround the `self._client.get_custom_models(error_map=error_map, **kwargs)` call